extensions/wikia/Wall/WallHelper.class.php

```diff
@@ -288,20 +288,19 @@ public function getWallComments( $parentId = null ) {
 	/**
 	 * @brief Gets wall comments data from memc/db
 	 *
-	 * @param array $comments an array with WallMessage instances
+	 * @param WallMessage[] $comments an array with WallMessage instances
 	 *
 	 * @return array
-	 * @author Andrzej 'nAndy' Åukaszewski
+	 * @author Andrzej 'nAndy' Łukaszewski
 	 */
-	private function getCommentsData( $comments ) {
+	private function getCommentsData( Array $comments ) {
 		wfProfileIn( __METHOD__ );
 
 		$timeNow = time();
 		$items = [ ];
 		$i = 0;
 		foreach ( $comments as $wm ) {
-			/** @var WallMessage $wm */
-			$data = $wm->getData( false, null, 30 );
+			$data = $wm->getData();
 
 			if ( !( $data['author'] instanceof User ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "author" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)